X16 Reference - 11 -  Working with CMDR-DOS.md

```diff
@@ -0,0 +1,368 @@
+# Chapter 11: Working With CMDR-DOS
+This manual describes Commodore DOS on FAT32, aka CMDR-DOS. 
+
+## CMDR-DOS
+
+Commander-16 duplicates and extends the programming interface used by Commodore's line of disk drives, including the 
+famous (or infamous) VIC-1541. CMDR-DOS uses the industry-standard FAT-32 format. Partitions can be 32MB up to 
+(in theory) 2TB and supports CMD-style partitions, subdirectories, timestamps and filenames up to 255 characters. 
+It is the DOS built into the [Commander X16](https://www.commanderx16.com).
+
+There are three basic interfaces for CMDR-DOS: the binary interface (SAVE,LOAD), the data file interface (OPEN, 
+PRINT#, INPUT#, GET#), and the command interface. We will give a brief summary of BASIC commands here, but please refer
+to the BASIC chapter for full syntax of each command. 
+
+If you are familiar with the SD2IEC or the CMD hard drive, navigating partitions and subdirectories is similar, with "CD", "MD", and "RD" commands
+to navigate directories.
+
+## Binary Load/Save
+
+The primary use of the binary interface is loading and saving program files and loading binary files into RAM. 
+
+Your binary commands are LOAD, SAVE, BLOAD, VLOAD, and BVLOAD. 
+
+This is a brief summary of the BASIC disk commands LOAD and SAVE. For full documentation, refer to Chapter 3: BASIC 
+Programming.
+
+### LOAD
+
+`LOAD <filename> [,device][,secondary_address][,start_address]`
+
+This reads a program file from disk. The first two bytes of the file are the memory location to which the file will be
+loaded, with the low byte first. BASIC programs will start with $01 $08, which translates to $0801, the start of BASIC
+memory. 
+The device number should be 8 for reading from the SD card. 
+
+secondary_address has multiple meanings: 
+
+* 0 or not present: load the data to address $0801, regardless of the address header. 
+
+* 1: load to the address specified 
+
+* 2: load into VERA RAM bank 0 (at the 16-bit address in the file)
+
+* 3: load into VERA RAM bank 1 (at the 16-bit address in the file)
+
+* 17 or $11: load into VERA RAM bank 16 or $0F 
```

Comment:
VERA only has banks 0 and 1, no?

Reply:
I pulled that example from somewhere else in the official docs.
I will remove, for now, but I'll probably add it back later once I confirm the bank numbers in VERA.